bin/src/report.dart

```diff
@@ -42,9 +42,9 @@ class ReportPart extends CommandLinePart {
 
     if (res.rest.length != 1) return print("Please specify a test file to run");
 
-    var pRoot = new Directory(res["package-root"]);
-    if (!pRoot.existsSync()) return print("Root directory does not exist");
-    log.info(() => "Package root is ${pRoot.absolute.path}");
+    FileSystemEntity pRoot = new File(res["packages"]);
```

Comment:
@Adracus @matanlurey thanks for the pointer to that package. Thing is, as explained in my comment on `upload.dart`, that the `coverage` package exclusively supports an option to provide a package file since v0.7.4 (i.e. package directories not supported anymore), and I don't see use in providing that-far-backwards compatibility.